internal/support/scheduler/utils.go

```diff
@@ -104,11 +106,12 @@ func parseInt64(value string) int64 {
 // Scheduler Queue Client
 var currentQueueClient *QueueClient // Singleton used so that queueClient can use it to de-reference readings
```

Comment:
(for new GitHub issue) My IDE indicates this global isn't referenced.  Should probably be removed.